Como encontrar patrocinadores (#13)
AfroPython/Manual-do-evento: patrocinadores/encontrar_patrocinadores.md

# Como encontrar Patrocinadores

Nós procuramos patrocinadores que são ativos na comunidade local de tecnologia, de startups até grandes corporações, incluindo organizações não governamentais(ONGs).

Além disso, empresas que não são da área de tecnologia podem ser abordadas: marcas de bebidas, restaurantes locais e etc. Todas podem ter interesse ou alguma relação com tecnologia.

O evento AfroPython deve ser sempre mantido como não lucrativo: Caso tenha sobra de dinheiro, deve ser utilizado para apoiar outros eventos como encontros do AfroPython, confraternização ou doado para a [Python Software Foundation](https://www.python.org/psf/) que sempre nos apoia nesses eventos. Não se esqueça que você não precisa de dinheiro, e que até uma pequena startup pode ajudar. Pode inclusive ser mais fácil der várias pequenas empresas patrocinando do que uma empresa gigantesca.

> 

Reviewer: Existe algum tipo de empresa que não gostariamos de contactar? Industria de armas, pornografica, ou que tenham algum histórico de racismo por exemplo

Reviewer 2: boa! talvez seja interessante algo como "contatar empresas que estejam de acordo com os valores explicitados no nosso código de conduta" ou algo assim

## O que você pode oferecer?

A oferta comum feita aos patrocinadores é que eles possam mostrar seus produtos ou serviços para as pessoas que estão participando do evento. Isso talvez inclua:

- logo, descrição e link no site
- banner no dia do evento
- brindes e folhetos distribuidos aos participantes durante o evento
- alguns minutos de fala durante o evento
- compartilhamento ou agradecimento público nas redes sociais
- e-mail enviado aos participantes através da organização do evento ( nunca dê o e-mail dos participantes )

Recomendamos perguntar aos patrocinadores sobre seus objetivos ao patrocinar o evento. Nós sempre tentamos colaborar com nossos parceiros e ter certeza que todos estamos atingindo nosso objetivos e entregando o valor esperado.

## Como encontrar patrocinadores?

Se você tiver um tempinho para isso, nós recomendamos fortemente a criação de um plano simples de patrocínio - Um PDF com algumas páginas. Diga quem é você, o que é o AfroPython e porque é incrível participar, quem vai vir ao evento e sobre valor para os patrocinadores ao apoiar o movimento.

> Você pode usar o [plano de patrocínio](https://github.com/DjangoGirls/resources/tree/master/For%20Sponsors) preparado por nós.

Reviewer: o link referencia o django girls, vamos manter ou adicionaremos o nosso?

Author: Não temos um nosso. Por enquanto podemos utilizar este até termos o nosso.

Os valores vão depender do seu orçamento para o evento, é comum que paguem por exemplo o almoço, ou as bebidas, lanche da tarde do evento. Não se esqueça toda contribuição é importante.

## Quem pode patrocinar?

- Procure por eventos parecidos na sua cidade como por exemplo, Django Girls e Rails Girls e entre em contato com as empresas que costumam apoiar esses eventos.
- Olhe no site dos últimos AfroPython e veja quais eventos e patrocinadores tem escritórios em sua cidade.
- Envie um e-mail ou ligue para empresas que você conhece.
- Compartilhe nas redes sociais uma chamada para patrocinadores. Assim as empresas sabem que podem patrocinar o seu evento e se juntar a você.
- Crie uma lista de empresas que utilizem Python e envie um e-mail para ver a possibilidade de patrocínio
- Entre em contato com empresas globais ou peça a [Django Software Foundation](https://djangoproject.com/) para ajudar. Tenha em mente que eles tem um orçamento limitado, então **só peça se você realmente precisa de ajuda**.

> - A [Python Software Foundation (PSF)](https://www.python.org/psf/) costuma fornecer \$25 dólares por participante para oficinas como essa e ele consideram que podem apoiar até 2 oficinas como essa por ano, a diferença precisa ser de pelo menos 6 meses entre os eventos. Tenha em mente que a PSF é uma organização sem fins lucrativos, antes de pedir ajuda, **peça a empresas locais**. Para pedir patrocínio a PSF siga as instruções [no site deles](https://www.python.org/psf/grants/).

Reviewer: ```suggestion
- A [Python Software Foundation (PSF)](https://www.python.org/psf/) costuma fornecer $25 dólares por participante para oficinas como essa e ele consideram que podem apoiar até 2 oficinas como essa por ano, a diferença precisa ser de pelo menos 6 meses entre os eventos. Tenha em mente que a PSF é uma organização sem fins lucrativos, antes de pedir ajuda, **peça a empresas locais**. Para pedir patrocínio a PSF siga as instruções [no site deles](https://www.python.org/psf/grants/).
```

# Como encontrar Patrocinadores

> Devido ao AfroPython ter como foco pessoas negras da região interessadas em tecnologia, nossos participantes são possíovels empregados e clientes de empresas que podem vir a patrocinar o evento.

Reviewer: ```suggestion
possíovels
```
possíveis

> Nós buscamos por patrocinadores que são ativos na comunidade local de tecnologia, de starups até grandes corporações mesmo organizações não governamentais(ONGs).

Reviewer: startups

Além disso, empresas que não são da área de tecnologia podem ser abordadas: marcas de bebidas, restaurantes locais e etc. Todas podem ter interesse ou alguma relação com tecnologia.

> AfroPython devem ser sempre mantido como não lucrativo: Caso tenha sobra de dinheiro, deve ser utilizado para apoiar outros eventos como encontros do AfroPython, confraternização ou doado para a [Python Software Foundation](https://www.python.org/psf/) que sempre nos apoia nesses eventos. Não se esqueça que você não precisa de dinheiro, e que até uma pequena startup pode ajudar. Pode inclusive ser mais fácil der várias pequenas empresas patrocinando do que uma empresa gigantesca.

Reviewer: ```suggestion
O evento AfroPython deve ser sempre mantido como não lucrativo: Caso tenha sobra de dinheiro, deve ser utilizado para apoiar outros eventos como encontros do AfroPython, confraternização ou doado para a [Python Software Foundation](https://www.python.org/psf/) que sempre nos apoia nesses eventos. Não se esqueça que você não precisa de dinheiro, e que até uma pequena startup pode ajudar. Pode inclusive ser mais fácil der várias pequenas empresas patrocinando do que uma empresa gigantesca.
```

## O que você não pode oferecer aos patrocinadores?

> O mais importante é: **nunca dê dados de nenhuma das pessoas envolvidads(nomes, e-mails, ou qualquer outra informação).** Nós somos muito comprometidos com a privacidade de cada pessoa. Você não deve quebrar esse regras nem que isso possa significar a perda de um patrocínio, brindes ou bom local para a oficina. Apenas não faça isso.

Reviewer: ```suggestion
O mais importante é: **nunca dê dados de nenhuma das pessoas envolvidas(nomes, e-mails, ou qualquer outra informação).** Nós somos muito comprometidos com a privacidade de cada pessoa. Essa regra não deve ser quebrada, nem que isso possa significar a perda de um patrocínio, brindes ou local para a oficina. Apenas não faça isso.
```

> Você pode enviar e-mails aos participantes sobre os patrocinadores, mas não pode compatilhar nenhuma informação sobreas pessoas envolvidas com patrocinadores.

Reviewer: ```suggestion
Você pode enviar e-mails aos participantes sobre os patrocinadores, mas não pode compatilhar nenhuma informação sobre as pessoas envolvidas com patrocinadores.
```

> É o que que patracinadores falem com as pessoas nos eventos desde que que não atrapalhe ou seja inconveniente para quem participa.

Reviewer: ```suggestion
E o que patrocinadores falem com as pessoas durante os eventos, desde que não seja inconveniente com quem participa.
```

> ## O que você pode oferece?

Reviewer: ```suggestion
## O que você pode oferecer?
```

> A oferta comum aos patrocinadores é que eles possam mostrar seus produtos ou serviçop para as pessoas que estão participando do evento. Isso talvez inclua:

Reviewer: ```suggestion
A oferta comum  feita aos patrocinadores é que eles possam mostrar seus produtos ou serviços para as pessoas que estão participando do evento. Isso talvez inclua:
```

- logo, descrição e link no site
- banner no dia do evento
- brindes e folhetos distribuidos aos participantes durante o evento

> - alguns minutos para falar durante o evento

Reviewer: ```suggestion
- alguns minutos de fala durante o evento
```

- compartilhamento ou agradecimento público nas redes sociais
- e-mail enviado aos participantes através da organização do evento, nunca dê o e-mail dos participantes

> Recomendamos falar com os patrocinadores e perguntas sobre seus objetivos ao patrocinar o evento. Nós sempre tentamos colaborar com nossos parceiros e ter certeza que todos estamos atingindo nosso objetivos e entregando o valor esperado.

Reviewer: ```suggestion
Recomendamos perguntar aos patrocinadores sobre seus objetivos ao patrocinar o evento. Nós sempre tentamos colaborar com nossos parceiros e ter certeza que todos estamos atingindo nosso objetivos e entregando o valor esperado.
```

## Como encontrar patrocinadores?

> Se você tiver um tempinho para isso, nós recomendamos fortemente a criação de um plano simples de patrocínio - Um PDF com algumas páginas. Diga quem é você, o que é o AfroPython e porque é incrível, quem vai vir ao evento e sobre o que de valor pode ter para os patrocinadores.

Reviewer: ```suggestion
Se você tiver um tempinho para isso, nós recomendamos fortemente a criação de um plano simples de patrocínio - Um PDF com algumas páginas. Diga quem é você, o que é o AfroPython e porque é incrível participar deste evento, quem vai vir ao evento e sobre quão a importância de patrocinar esse evento.
```

Você pode usar o [plano de patrocínio](https://github.com/DjangoGirls/resources/tree/master/For%20Sponsors) preparado por nós.

Os valores vão depender so seu orçamento para o evento, é comum que paguem por exemplo o almoço, ou as bebidas, lanche da tarde do evento. Não se esqueça toda contribuição é importante.

## Quem pode patrocinar?

> - Procure pod eventos parecidos nas sua cidade com Django Girls e Rails Girls e entre em contato com as empresas que costumam apoiar esses eventos.

Reviewer: ```suggestion
- Procure por eventos parecidos nas sua cidade como por exemplo, Django Girls e Rails Girls e entre em contato com as empresas que costumam apoiar esses eventos.
```

- Olhe no site dos últimos AfroPython e veja quais eventos e patrocinadores tem escritórios em sua cidade.
- Envie um e-mail ou ligue para empresas que você conhece.
- Compartilhe nas redes sociais uma chamada para patrocinadores. Assim as empresas sabem que podem patrocinar o seu evento e se juntar a você.
- Crie uma lista de empresas que utilizem Python e envie um e-mail para ver a possibilidade de patrocínio
- Entre em contato com empresas globais ou peça a [Django Software Foundation](https://djangoproject.com/) para ajudar. Tenha em mente que eles tem um orçamento limitado, então **só peça se você realmente precisa de ajuda**.

> - A [Python Software Foundation (PSF)](https://www.python.org/psf/) costuma fornecer \$25 dólares por participante para oficinas como essa e ele consideram que podem apoiar até 2 oficinas como essa por ano, a diferença precisa ser de pelo meno 6 meses entre um e outro. Tenha em mente que a PSF é uma organização sem fins lucrativos, antes de pedir ajuda, **tente explorar as empresas locais**. Para pedir patrocínio a PSF siga as intruções [no site deles](https://www.python.org/psf/grants/).

Reviewer: ```suggestion
- A [Python Software Foundation (PSF)](https://www.python.org/psf/) costuma fornecer \$25 dólares por participante para oficinas como essa e ele consideram que podem apoiar até 2 oficinas como essa por ano, a diferença precisa ser de pelo menos 6 meses entre os eventos. Tenha em mente que a PSF é uma organização sem fins lucrativos, antes de pedir ajuda, **peça a empresas locais**. Para pedir patrocínio a PSF siga as instruções [no site deles](https://www.python.org/psf/grants/).
```
